src/Router.jsx

```diff
@@ -0,0 +1,53 @@
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+
+import NotFound from './pages/NotFound';
+import Home from './pages/Home';
+import About from './pages/About';
+import Account from './pages/account/Account';
+import SavedBooks from './pages/account/SavedBooks';
+import Messages from './pages/account/Messages';
+import Orders from './pages/account/Orders';
+import Profile from './pages/account/Profile';
+import ProfileEdit from './pages/account/ProfileEdit';
+import Books from './pages/books/Books';
+import BookEdit from './pages/books/BookEdit';
+import BookNew from './pages/books/BookNew';
+import Cart from './pages/cart/Cart';
+import CheckOut from './pages/cart/CheckOut';
+import PasswordReset from './pages/auth/PasswordReset';
+import PasswordEdit from './pages/auth/PasswordEdit';
+import SignIn from './pages/auth/SignIn';
+import SignUp from './pages/auth/SignUp';
+import Header from './layouts/Header';
+import Footer from './layouts/Footer';
+
+const Router = () => {
+  return (
+    <BrowserRouter>
+      <Header />
+      <Routes>
+        <Route path="*" element={<NotFound />} />
```

Comment:
Same here. Routes could be grouped the same way as imports. Example:
```
{/* Public Routes */}
...
{/* Account Routes */}
...
{/* Books Routes */}
...
{/* Cart Routes */}
...
{/* Auth Routes */}
...
{/* 404 Not Found */}
```
This is not necessary at the moment but something to consider.